install/helm/agones/templates/admissionregistration.yaml

```diff
@@ -18,10 +18,40 @@
 {{- $cert := genSignedCert $cn nil (list $altName1 $altName2) 3650 $ca }}
 {{- if .Values.agones.registerWebhooks }}
 ---
+apiVersion: apiregistration.k8s.io/v1beta1

```

Comment:
I put the APIService here, because it uses the same cert - but I don't know if "registerWebhooks" is still accurate, or if the file name "admissionregistration" is still accurate.

Thoughts on what should be done here?

Iinitial idea: rename files to "extensions.yaml" or something similiar. Maybe add an extra flag for the APIService that is seperate from the webhooks? agones.registerAPIService` ?

Reply:
I'm wondering why `registerWebhooks` is a parameter, if you don't Agones won't work ?

I think if you just rename the file to `extensions.yaml` that would do it.

Reply:
basically - because people sometimes have weird requirements for installations, and need an escape hatch :man_shrugging: 

The escape hatch it to disable a certain section, and then manually install the thing that's been turned off in the special way that their platform requires.

That's really about it -- have run into it a few times with some users I've worked with 1:1 where they have needed this for things i wouldn't otherwise have thought of - so better to have and not want, than not have and be blocked until next release.

Reply:
Fixed! moved to extensions.yaml, and made a new flag for the apiservice.